cas_implementations/src/path.rs

```diff
@@ -11,9 +11,12 @@ pub fn storage_path(path: &Path, folder_name: &str) -> HcResult<String> {
 
 pub fn create_path_if_not_exists(path: &str) -> HcResult<()> {
     if !Path::new(path).exists() {
-        return DirBuilder::new()
-            .create(path)
-            .map_err(|_| HolochainError::IoError("Could not create directory".to_string()));
+        return DirBuilder::new().recursive(true).create(path).map_err(|e| {
```

Comment:
can we use `?` instead of `map_err` here?

Reply:
no because I want to create a much more informative error.  The ? passes back something with no contextual info.